Shopify/dawn
# Theme check manual fixes

**Why are these changes introduced?**

This PR is part 2/2 for #1364 

**What approach did you take?**

Fixed all of the following theme check warnings:

- **10** for `` DeprecatedFilter: Deprecated filter `img_url`, consider using an alternative: `image_url` ``
- **8** for `ImgLazyLoading`
- **2** for `LiquidTag: Use {% liquid ... %} to write multiple tags`

And added the necessary `theme-check-disable` comments.

![image](https://user-images.githubusercontent.com/28404165/154762971-a0d502ab-bb13-49c8-a956-1d5d0293a529.png)

**Other considerations**

There are a few places where we don't want lazy loading added and `{% comment %}theme-check-disable ImgLazyLoading{% endcomment %}` should be added to these files if they contain lazy load attribute conditions to prevent `theme check` warnings:

- **Collection banner**
- **Product page**: Main image
- **Collection page**: First 2 images
- **Search page**: First 2 images

**CC**: @tyleralsbury 

- [Disable checks with comments](https://github.com/Shopify/theme-check#disable-checks-with-comments) README

**Testing steps/scenarios**
- [ ] Look at each of the files changes and ensure the following was fixed:
  - **10** warnings for deprecated `img_url`: these couldn't be auto-corrected due to `{% assign %}` variables.
  - **8** for `ImgLazyLoading`: the `theme-check-disable ImgLazyLoading` comment was added to the respective files.
  - **2** for `LiquidTag: Use {% liquid ... %} to write multiple tags.`: 2 instances in `main-product.liquid`.
- [ ] You can also make sure that running `shopify theme check` or `theme-check` returns:
  - `233 files inspected, 0 offenses detected, 0 offenses auto-correctable` 

**Demo links**

- [Store](https://os2-demo.myshopify.com/?preview_theme_id=127615205398)
- [Editor](https://os2-demo.myshopify.com/admin/themes/127615205398/editor)

**Checklist**
